packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -361,10 +361,12 @@ class RCTAztecView: Aztec.TextView {
     }
 
     public override func insertDictationResult(_ dictationResult: [UIDictationPhrase]) {
-        let objectPlaceholder = "\u{FFFC}"
```

Comment:
The new approach to adding text doesn't present the problem with `obj` remaining in dictated text.